src/container/srv_target.c

```diff
@@ -1579,8 +1579,8 @@ ds_cont_tgt_force_close(uuid_t cont_uuid)
 {
 	int rc;
 
-	D_DEBUG(DF_DSMS, DF_CONT": Force closing all handles for container "
-		DF_UUID"\n", DP_CONT(NULL, NULL), cont_uuid);
```

Comment:
This is the cause of the most recent error, we were logging a raw uuid as a string.  This code is about five months old, so I can only conclude something else has landed elsewhere where we're now taking this code-path where we weren't before.